sdk/react/bottomOverlay.js

```
@@ -117,13 +117,20 @@ var BottomOverlay = React.createClass({
     var topOffset = this._renderTopOffset(scrubberSize);
     var leftOffset = this._renderLeftOffset(scrubberSize, percent, progressBarWidth);
     var positionStyle = {top:topOffset, left:leftOffset};
+    var scrubberStyle = this._customizeScrubber();
 
     return (
-      <View style={[styles.progressScrubber, positionStyle,{width:scrubberSize, height:scrubberSize}]}>
+      <View style={[scrubberStyle, positionStyle,{width:scrubberSize, height:scrubberSize}]}>
       </View>
       );
   },
 
+  _customizeScrubber: function() {
+    var scrubberStyle = {flex: 0, position: "absolute", backgroundColor: this.props.config.controlBar.scrubberBar.scrubberHandleColor, 
```

In retrospect, this will break when people use an old "Skin.json". file.  

Is there any chance we can log an error  in react-native, if scrubberHandleColor and scrubberHandleBorderColor do not exist, we can raise an error, saying "controlBar.scrubberBar.scrubberHandleBorderColor not defined in your skin.json. update your skin.json file to the latest provided file, or add this to your skin.json"?  I don't know if we have any historical precedent for this, but this will happen often for people and we should have some warning

---

Another idea is to set default color values if scrubberHandleColor and scrubberHandleBorderColor are not defined.

---

I checked in a fix that logs error messages, but that will generate a lot of messages since it will log every time the scrubber is rendered.